<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUp</class>
 <widget class="QWidget" name="SignUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>731</width>
     <height>591</height>
    </rect>
   </property>
   <property name="title">
    <string>打卡</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>356</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="time_label">
         <property name="text">
          <string>实时时间：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="time_lineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="eventName_label">
         <property name="text">
          <string>赛事名称：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="eventName_comboBox"/>
       </item>
       <item>
        <widget class="QLabel" name="checkPointName_label">
         <property name="text">
          <string>打卡点名称：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="checkPoint_comboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="Time_label">
         <property name="text">
          <string>时间戳：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTimeEdit" name="Time_timeEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="btn_Time">
         <property name="text">
          <string>记录</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="direction_label">
         <property name="text">
          <string>方向：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="direction_lineEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="direction_label_2">
         <property name="text">
          <string>0表示到达，1表示离开</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="sumTime_label">
         <property name="text">
          <string>总用时：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="sumTime_lineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="rank_label">
         <property name="text">
          <string>名次：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="rank_lineEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>367</x>
      <y>30</y>
      <width>361</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="rfidTag_label">
         <property name="text">
          <string>卡号：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="rfidTag_lineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="btn_recognize">
         <property name="text">
          <string>识别</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_refresh">
         <property name="text">
          <string>刷新打卡表</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_reset">
         <property name="text">
          <string>重置</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>711</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
